puppet/modules/slave/manifests/init.pp

```diff
@@ -162,6 +162,19 @@
     }
   }
 
+  # Needed by foreman_openscap gem dependency OpenSCAP
+  if $osfamily == 'RedHat' {
+    yumrepo { 'isimluk-openscap':
+      enabled  => 1,
+      gpgcheck => 0,
+      baseurl  => "http://copr-be.cloud.fedoraproject.org/results/isimluk/OpenSCAP/epel-${::operatingsystemmajrelease}-\$basearch/",
+      includepkgs => ['openscap'],
+    } ->
+    package { 'openscap':
+      ensure => '1.2.3',
```

Comment:
This needs to be the full VR, so `1.2.3-0.1.el6` etc or just use `latest` (preferred).